compiler/rustc_errors/src/lib.rs

```diff
@@ -1548,7 +1549,19 @@ pub fn add_elided_lifetime_in_path_suggestion(
 
 /// Useful type to use with `Result<>` indicate that an error has already
 /// been reported to the user, so no need to continue checking.
-#[derive(Clone, Copy, Debug, Encodable, Decodable, Hash, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(
```

Comment:
nit: this probably should be split into two derives so it wraps more neatly

Reply:
Yes, that's better.